src/retasc/product_pages_api.py

```diff
@@ -0,0 +1,140 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+"""
+Production Pages API
+"""
+
+from functools import lru_cache
+
+from retasc import constants, requests_session
+
+
+class ProductPagesApi:
+    """
+    Product Pages API Client
+    """
+
+    def __init__(self, api_url: str):
+        self.api_url = api_url
+        self.session = requests_session.requests_session()
+
+    @lru_cache(maxsize=None)
+    @staticmethod
```

Comment:
These are not static methods since they use class instance members (it takes `self` argument).

Reply:
sorry, forgot to remove, they were static but not anymore.